react/src/lib/Slider/index.js

```diff
@@ -155,10 +155,24 @@ class Slider extends React.Component {
     };
 
     return (
-      <div className={
-        `md-slider ${className}` +
-        `${(disabled && ` md-slider--disabled`) || ''}` +
-        `${(className && ` ${className}`) || ''}`
+      <div 
+        className={
+          `md-slider ${className}` +
```

Comment:
Agree with removing class name here as well because it also adds unnecessary spaces.